internal/service/location/controller.go

```diff
@@ -194,5 +221,5 @@ func (c *Controller) QueryLocation(ctx context.Context, request *locationV1beta1
 			},
 		}, nil
 	}
-	return nil, fmt.Errorf("location not found for WorkerId=%s", request.WorkerId)
+	return nil, status.Errorf(codes.NotFound, "WorkerId: %s", request.WorkerId)
```

Comment:
nit: My preference here is that the error message should be something static (like "Worker not found") and high-cardinality fields (like IDs) should be kept in error metadata. (See #64)